glide.yaml

```diff
@@ -38,7 +38,8 @@ import:
 
 # openshift second
 - package: github.com/openshift/api
-  version: master
+  repo:    https://github.com/deads2k/api.git
```

Comment:
I hope you're not going to merge this as this.
/hold

Reply:
> I hope you're not going to merge this as this.
> /hold

it won't merge with bad travis